packages/astro/src/@types/astro.ts

```diff
@@ -927,46 +925,6 @@ export interface AstroUserConfig {
 	 * These flags are not guaranteed to be stable.
 	 */
 	experimental?: {
-		/**
```

Comment:
One thing to note is that removing these will remove them from docs too — https://docs.astro.build/en/reference/configuration-reference/#experimental-flags

Maybe that doesn’t really matter, but we should make sure to say v2-beta required when documenting `prerender` (error overlay’s probably less important docs-wise). I don’t think we got a `prerender` docs PR yet, right @natemoo-re?

Reply:
Correct, will let you know when that's ready!

Reply:
https://github.com/withastro/docs/pull/2279